src/plugins/orangehrmDashboardPlugin/Dao/ChartDao.php

```diff
@@ -130,6 +130,7 @@ public function getEmployeeDistributionByLocation(): array
         $q->andWhere($q->expr()->isNull('employee.purgedAt'));
         $q->addGroupBy('location.id');
         $q->addOrderBy('COUNT(employee.empNumber)', 'DESC');
+        $q->addOrderBy('location.name', 'ASC');
```

Comment:
```suggestion
        $q->addOrderBy('location.name', \OrangeHRM\ORM\ListSorter::ASCENDING);
```
prefer if you can use this const

Reply:
Add to https://github.com/orangehrm/orangehrm/pull/1440